templates/aws-stack.yml

```diff
@@ -1232,7 +1237,7 @@ Resources:
     Properties:
       Location:
         ApplicationId: arn:aws:serverlessrepo:us-east-1:172840064832:applications/buildkite-agent-scaler
```

Comment:
Is it worth parameterising `ApplicationId` as well if people want to use forks of the scaler?

(no is a perfectly acceptable answer)

Reply:
Good call. I believe it's a bit complicated to set up their CI to publish to serverless repo, but I guess if they fork our repo they should be able to figure it out.